akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowExpandSpec.scala

```diff
@@ -22,6 +21,9 @@ class FlowExpandSpec extends AkkaSpec {
   "Expand" must {
 
     "pass-through elements unchanged when there is no rate difference" in {
+      // Shadow the fuzzed materializer (see the ordering guarantee needed by the for loop below).
+      implicit val materializer = ActorMaterializer(settings.withFuzzing(false))
```

Comment:
Why is it a setting rather than a type of Materializer? Now we're requiring all Materializers to respect this setting, right?